frontend/src/metabase/setup/components/DatabaseStep/DatabaseStep.tsx

```diff
@@ -127,25 +128,46 @@ const DatabaseForm = ({
       formName="database"
       database={database}
       onSubmit={handleSubmit}
+      submitTitle={t`Connect database`}
     >
       {({
         Form,
         FormField,
-        FormFooter,
+        FormSubmit,
         formFields,
         values,
         onChangeField,
+        submitTitle,
+        error,
       }: FormProps) => (
         <Form>
           <FormField name="engine" onChange={handleEngineChange} />
           <DriverWarning
             engine={values.engine}
             onChange={engine => onChangeField("engine", engine)}
           />
+          <FormError
+            className="mt3 mb4"
```

Comment:
I see, so generally, utility class name is discouraged right? Then is there a case where we would use utility classes?

Reply:
also, `space(number)` has a limited number of spaces to choose from. What if I need `24px` (as in the design). The 

`SPACE_LEVELS` is also a form designed system, but this isn't actually in sync with the actual space in our design system
https://github.com/metabase/metabase/blob/c9949a16e4d2e82d077b0fb265c8fe085cf27e94/frontend/src/metabase/styled-components/theme/constants.ts#L1

![image](https://user-images.githubusercontent.com/1937582/170206558-1f788938-0e51-4365-a325-9800e59aa2d6.png)

Maybe this should be addressed another time as it would affect all places that use `space(number)`, and I should use the exact value for now?

e.g. 
```css
margin-top: 24px
margin-bottom: 32px
```

What do you think?